libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/mpp/apple/AppleXcodeTasks.kt

```diff
@@ -200,4 +202,15 @@ private val Framework.namePrefix: String
     )
 
 private fun Project.appleFrameworkDir(frameworkSearchDir: File) =
-    buildDir.resolve("xcode-frameworks").resolve(frameworkSearchDir)
\ No newline at end of file
+    buildDir.resolve("xcode-frameworks").resolve(frameworkSearchDir)
+
+private fun Project.copyFiles(files: Array<File>, destDir: File) {
```

Comment:
It would be perfect to add comment about symlinks behavior with YouTrack issue ID.